methods_lab.js

```diff
@@ -0,0 +1,238 @@
+/*1. Given an array of numbers, write a function that returns a new array where every element is incremented by 10.
```

Comment:
Great job! I see that you understand array methods. The only thing that you should work on is cleaner code. For the  first problem I demonstrated how it could be made cleaner. You can actually utilize it in all your code.